File: app/src/main/java/forpdateam/ru/forpda/api/search/interfaces/ISearchItem.java
package forpdateam.ru.forpda.api.search.interfaces;

import forpdateam.ru.forpda.api.IBaseForumPost;

/**
 * Created by radiationx on 27.04.17.
 */

public interface ISearchItem extends IBaseForumPost {
    String getImageUrl();

    String getTitle();

    String getDesc();
}

File: app/src/main/java/forpdateam/ru/forpda/entity/remote/search/ISearchItem.kt
package forpdateam.ru.forpda.entity.remote.search

import forpdateam.ru.forpda.entity.remote.IBaseForumPost

/**
 * Created by radiationx on 27.04.17.
 */

interface ISearchItem : IBaseForumPost {
    val imageUrl: String?
    val title: String?
    val desc: String?
}
